src/Compilers/CSharp/Test/Syntax/Parsing/UTF8StringLiteralsParsingTests.cs

```diff
@@ -520,5 +588,353 @@ public void Interpolation_04()
+                        N(SyntaxKind.UTF8StringLiteralToken, "@\"\"\"hello\"\"\"" + suffix);
+                    }
+                    EOF();
+                }
+            }
+        }
+
+        [Theory]
+        [InlineData("u8")]
+        [InlineData("U8")]
+        public void Errors_17(string suffix)
+        {
+            // The behavior is consistent with how type suffixes are handled on numeric literals, see Errors_06.
+            UsingExpression(@"""""""hello"""""" " + suffix,
+                // (1,1): error CS1073: Unexpected token 'u8'
+                // """hello""" u8
+                Diagnostic(ErrorCode.ERR_UnexpectedToken, @"""""""hello""""""").WithArguments(suffix).WithLocation(1, 1)
+                );
+
+            N(SyntaxKind.StringLiteralExpression);
+            {
+                N(SyntaxKind.SingleLineRawStringLiteralToken, "\"\"\"hello\"\"\"");
+            }
+            EOF();
+        }
+
+        [Theory]
+        [InlineData("u")]
+        [InlineData("U")]
+        public void Errors_18(string suffix)
+        {
+            UsingExpression(@"""""""hello""""""" + suffix,
+                // (1,1): error CS1073: Unexpected token 'u'
+                // """hello"""u
+                Diagnostic(ErrorCode.ERR_UnexpectedToken, @"""""""hello""""""").WithArguments(suffix).WithLocation(1, 1)
+                );
+
+            N(SyntaxKind.StringLiteralExpression);
+            {
+                N(SyntaxKind.SingleLineRawStringLiteralToken, "\"\"\"hello\"\"\"");
+            }
+            EOF();
+        }
+
+        [Fact]
+        public void Errors_19()
+        {
+            UsingExpression(@"""""""hello""""""8",
+                // (1,1): error CS1073: Unexpected token '8'
+                // """hello"""8
+                Diagnostic(ErrorCode.ERR_UnexpectedToken, @"""""""hello""""""").WithArguments("8").WithLocation(1, 1)
+                );
+
+            N(SyntaxKind.StringLiteralExpression);
+            {
+                N(SyntaxKind.SingleLineRawStringLiteralToken, "\"\"\"hello\"\"\"");
+            }
+            EOF();
+        }
+
+        [Theory]
+        [InlineData("u80")]
+        [InlineData("U80")]
+        public void Errors_20(string suffix)
+        {
+            // The behavior is consistent with how type suffixes are handled on numeric literals, see Errors_05.
+            UsingExpression(@"""""""hello""""""" + suffix,
+                // (1,1): error CS1073: Unexpected token '0'
+                // """hello"""U80
+                Diagnostic(ErrorCode.ERR_UnexpectedToken, @"""""""hello""""""" + suffix.Substring(0, 2)).WithArguments("0").WithLocation(1, 1)
+                );
+
+            N(SyntaxKind.UTF8StringLiteralExpression);
+            {
+                N(SyntaxKind.UTF8SingleLineRawStringLiteralToken, "\"\"\"hello\"\"\"" + suffix.Substring(0, 2));
+            }
+            EOF();
+        }
+
+        [Theory]
+        [InlineData("u80")]
+        [InlineData("U80")]
```

Comment:
Should these be `"u8"` and `"U8"` rather than `"u80"` and `"U80"`? Same question for other `Interpolation_0*()` tests below.

Reply:
> Should these be "u8" and "U8" rather than "u80" and "U80"?

Yes, good catch. I'll fix.